chapters/annotations.tex

```diff
@@ -1752,8 +1752,8 @@ \subsubsection{Conversion Rules}\label{conversion-rules}
 Conversions should be applied in all places where named element are used in code, including Modelica URIs (for example, in \lstinline!Documentation! annotations).
 \end{nonnormative}
 
-These functions can be called with literal strings or array of strings
-and vectorize according to \cref{scalar-functions-applied-to-array-arguments}.
+These functions can be called with literal strings or arrays of strings and vectorize according to \cref{scalar-functions-applied-to-array-arguments}.
```

Comment:
I think there must be some missing rules for the arrays of strings.  I get the impression that both `{"a"}` and `fill("", 0)` are examples of allowed arrays, but what about the following?
- `{"a" + "b"}
- `{String(true)}`

Could it be that the intention is to say that the only allowed kind of string expression is string literals, and that arrays of strings may be constructed using `fill` and `array` (possibly as `{…}`) without iterators.

Are there good use cases for allowing `fill` to create non-empty arrays?